ntbs-service/Models/SeedData/Continents.cs

```diff
@@ -27,7 +27,7 @@ public static IEnumerable<Continent> GetContinents()
                 new Continent { ContinentId = 16, Name = "South East Asia", Code = "SEA" },
                 new Continent { ContinentId = 17, Name = "Sub-Saharan Africa", Code = "SSA" },
                 new Continent { ContinentId = 18, Name = "Unknown", Code = "UU" },
-                new Continent { ContinentId = 19, Name = "West Europe", Code = "WEU "}
+                new Continent { ContinentId = 19, Name = "West Europe", Code = "WEU" }
```

Comment:
This blank space snuck into NTBS-3069, but not the actual migration that ticket generated. Fixing it here so that the migration in this ticket doesn't try to alter this row.